### PR TITLE
QF-1546 - Added UpdateAparSummary function

### PR DESCRIPTION
**Pull request by PeterDavidWilliams**

### CODE REVIEW

File: src/SFA.DAS.Assessor.Functions.ExternalApis/Assessor/IAssessorServiceApiClient.cs [hunk 1]
```diff
@@ -33,5 +33,6 @@ public interface IAssessorServiceApiClient : IApiClientBase
         Task RebuildExternalApiSandbox();
         Task ImportLearners();
         Task RefreshProviders();
+        Task UpdateAssessmentOrganisationsList();
```
Comment (ChrisJWoodcock) [inline src/SFA.DAS.Assessor.Functions.ExternalApis/Assessor/IAssessorServiceApiClient.cs:36]: Should be UpdateAparSummary

**Review by ChrisJWoodcock — CHANGES_REQUESTED** [review]

File: src/SFA.DAS.Assessor.Functions/Domain/Assessors/AssessmentOrganisationsListUpdateCommand.cs [hunk 2]
```diff
@@ -0,0 +1,26 @@
+﻿using Microsoft.Extensions.Logging;
+using SFA.DAS.Assessor.Functions.ExternalApis.Assessor;
+using SFA.DAS.Assessor.Functions.Functions.Assessors;
+using System.Threading.Tasks;
+
+namespace SFA.DAS.Assessor.Functions.Domain.Assessors
+{
+    public class AssessmentOrganisationsListUpdateCommand
+    {
```
Comment (SarahDuncan) [inline src/SFA.DAS.Assessor.Functions/Domain/Assessors/AssessmentOrganisationsListUpdateCommand.cs:9]: naming

File: src/SFA.DAS.Assessor.Functions/Domain/Assessors/Interfaces/IAssessmentOrganisationsListUpdateCommand.cs [hunk 3]
```diff
@@ -0,0 +1,8 @@
+﻿using SFA.DAS.Assessor.Functions.Domain.Interfaces;
+
+namespace SFA.DAS.Assessor.Functions.Domain.Assessors.Interfaces
+{
+    public interface IAssessmentOrganisationsListUpdateCommand : ICommand
+    {
```
Comment (SarahDuncan) [inline src/SFA.DAS.Assessor.Functions/Domain/Assessors/Interfaces/IAssessmentOrganisationsListUpdateCommand.cs:6]: naming

File: src/SFA.DAS.Assessor.Functions/Functions/Assessors/AssessmentOrganisationsListUpdateFunction.cs [hunk 4]
```diff
@@ -0,0 +1,42 @@
...
+    public class AssessmentOrganisationsListUpdateFunction
+    {
+        private readonly IAssessmentOrganisationsListUpdateCommand _command;
+
+        public AssessmentOrganisationsListUpdateFunction(IAssessmentOrganisationsListUpdateCommand command)
+        {
+            _command = command;
+        }
+
+        [FunctionName("AssessmentOrganisationsListUpdate")]
```
Comment (SarahDuncan) [inline src/SFA.DAS.Assessor.Functions/Functions/Assessors/AssessmentOrganisationsListUpdateFunction.cs:18]: should we change the function name

**Review by ChrisJWoodcock — APPROVED** [review]

**Pull request merged**